index.html

```diff
@@ -48,5 +48,157 @@ <h1 class="title">I’m Mariane<span id="info" class="title">Glad to see you!</s
+                            See Project
+                        </button>
+                    </div>
+                </li>
+            </ul>
+            
+            <ul>
+                <li class="project">
+                    <img src="src/Snapshoot-Portfolio2.png" alt="Snapshot2">
+                    <div class="description">
+                        <h2 class="workstitle">Tonic</h2>
+                        <div class="info">
+                            <p class="subTitle">Canopy</p>
+                            <ul class="infoContainer">
+                                <li class="role">Back End Dev</li>
+                                <li class="year">2015</li>
+                            </ul>
+                        </div>
+                        <p class="projectDescription">A daily selection of privately personalized reads;
+                             no accounts or sign-ups required.                  
+                        </p>
+                        <div class="tags">
+                            <ul class="tagsFlex">
+                                <li class="html">HTML</li>
+                                <li class="css">CSS</li>
+                                <li class="javascript">Javascript</li>
+                            </ul>
+                        </div>
+                        <button type="button" class="viewProject">
+                            See Project
+                        </button>
+                    </div>
+                </li>
+            </ul>
+
+            <ul>
+                <li class="project">
+                    <img src="src/Snapshoot-Portfolio3.png" alt="Snapshot3">
+                    <div class="description">
+                        <h2 class="workstitle">Multi-Post Stories</h2>
+                        <div class="info">
+                            <p class="subTitle">Canopy</p>
+                            <ul class="infoContainer">
+                                <li class="role">Back End Dev</li>
+                                <li class="year">2015</li>
+                            </ul>
+                        </div>
+                        <p class="projectDescription">A daily selection of privately personalized reads;
+                             no accounts or sign-ups required.                  
+                        </p>
+                        <div class="tags">
+                            <ul class="tagsFlex">
+                                <li class="html">HTML</li>
+                                <li class="css">CSS</li>
+                                <li class="javascript">Javascript</li>
+                            </ul>
+                        </div>
+                        <button type="button" class="viewProject">
+                            See Project
+                        </button>
+                    </div>
+                </li>
+            </ul>
+    
+        </div>
+    </section>
```

Comment:
- [ ] Please make sure your design aligns with the original design guidelines(fonts, colors, margin, padding, etc)
- [ ] The button should look the same as the original design and with the same dimensions
- [ ] Your cards are a little bit too wide which is causing horizontal scrolling. Kindly prevent this by making sure you stick to design dimensions.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/153704260-158093d9-9333-4d45-904d-3fc6c032e8cf.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/153704290-d3f8c12a-987a-43de-b41a-df3cd069924e.png)